parser/src/aspects/call.rs

```diff
@@ -58,6 +64,54 @@ impl<'a> Parser<'a> {
             _ => self.literal(),
         }
     }
+
+    fn call_type_parameters(&mut self) -> ParseResult<Vec<&'a str>> {
+        let start = self.cursor.peek();
+        if start.token_type != SquaredLeftBracket {
+            return Ok(Vec::new());
+        }
+
+        self.cursor.next()?; //skip start (']') peeked token.
+
+        if self
+            .cursor
+            .advance(spaces().then(of_type(SquaredRightBracket)))
+            .is_some()
+        {
+            return Err(self.mk_parse_error(
+                "unexpected empty type parameter list",
+                start..self.cursor.peek(),
+                ParseErrorKind::Unexpected,
+            ));
+        }
+        let mut tparams = vec![self.call_type_parameter()?];
+
+        while self
+            .cursor
+            .advance(spaces().then(of_type(SquaredRightBracket)))
```

Comment:
```suggestion
            .advance(word_seps().then(of_type(SquaredRightBracket)))
```
`spaces()` don't handle escaped new lines.